src/Commands/ThemeResetLayout.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace FOP\Console\Commands;
+
+use FOP\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class ThemeResetLayout extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('fop:theme-reset')
+            ->setDescription('Reset current theme layout')
+            ->setHelp('Disable & re-enable theme configuration')
+            ->addArgument('theme', InputArgument::OPTIONAL, 'Theme on which the action will be executed');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        $theme = $input->getArgument('theme');
+        $container = $this->getContainer();
+
+        if (!$theme) {
+            $io->text('(?) No theme selected, trying to select current theme...');
```

Comment:
maybe ask a confirmation before actually do the reset in that case of select auto default theme ?

Reply:
@zalexki I don't think so : in this case there is only one theme, no need to add this IMO.

Reply:
ye ok, that's a user experience you have to choose of course :)